backend/src/sync/mod.rs

```diff
@@ -74,6 +76,26 @@ pub(crate) struct SyncConfig {
     /// load on Opencast, increase to speed up download a bit.
     #[config(default = 8)]
     concurrent_download_tasks: u8,
+
+    /// List of deletion modes that which, if any, realm pages are to be deleted
+    /// automatically when the corresponding Opencast item (series, event or playlist)
+    /// is deleted.
+    /// If configured, Tobira will delete the corresponding realm page(s) when they meet
+    /// the following conditions:
+    /// - Realm name is derived from the deleted item.
+    /// - Realm has no sub realms.
+    /// - Realm has no other blocks than the deleted item.
+    ///
+    /// The last option can be disabled by adding `:eager` to the deletion mode.
+    ///
+    /// Example:
+    /// ```
+    /// auto_delete_pages = ["series", "events:eager"]
+    /// ```
+    ///
+    /// This would delete series pages in non-eager mode and event pages in eager mode.
+    #[config(default = [])]
+    pub auto_delete_pages: Vec<DeletionMode>,
```

Comment:
Oh, it might be nice to add a validation here to make sure each kind of item is only present once? See `validate_logos` for how this can be done in terms of confique attributes.